scripts/HPC_chatbot_preprocessor/parsed_mds/generic/compiling_your_software/compiling_your_software_paragraph_1.txt

```diff
@@ -0,0 +1,10 @@
+Compiling and testing your software on the HPC
+All nodes in the HPC cluster are running the "RHEL 8.8 (accelgor, doduo, donphan, gallade, joltik, skitty)" 
```

Comment:
what happened with the double quotes?, not needed to include the clusters i think